src/content/admin/dashboard.mdx

```diff
@@ -207,39 +198,6 @@ From this page you can follow a simple step-by-step guide to get the major compo
 
 ## Registry Credentials
 
-In this section of the Admin dashboard you can manage the private registry credentials for your Okteto instance. The credentials you configure here are injected by Okteto into any operation (e.g. build, deploy) performed by developers so they don't need direct access to the credentials. Once you configure the credentials here all developers in your Okteto instance will be able to use the respective registries while developing with Okteto. You can also manage these credentials directly in your Kubernetes cluster using the [CRDs](self-hosted/manage/crds.mdx#private-registries) created by Okteto.
-
-If the setting [`daemonset.configurePrivateRegistriesInNodes.enabled`](self-hosted/helm-configuration.mdx#daemonset) is enabled, Okteto's daemonset will also configure these credentials in the nodes so the cluster can also pull images from those registries.
+If you choose to use your own registry service instead of [Okteto's Container Registry](core/container-registry.mdx), you can configure credentials for that service here.
```

Comment:
It's not one or the other. People must use Okteto Registry in any case. Private registries are to pull private images from other private registries.

Reply:
Oh I understand now. Let up update this copy

Reply:
Does this sound ok? `If you choose to pull images from a private registry in addition to [Okteto's Container Registry](core/container-registry.mdx), you can configure credentials for that service here.`

Reply:
LGTM